recipe/build-mlflow.bat

```diff
@@ -13,6 +13,13 @@ if [%PKG_NAME%] == [mlflow-skinny] (
   copy pyproject.full.toml pyproject.toml
 )
 
+if [%PKG_NAME%] == [mlflow-ui] (
+  pushd mlflow\server\js
+  yarn install
```

Comment:
Is the build environment configured to use node16? 

Should this be validated explicitly?

``` bash
where nvm > nul 2>&1
if %errorlevel% neq 0 (
    echo Error: nvm is not installed or not in the system path.
    exit /b 1
)
nvm install 16
nvm use 16
pushd mlflow\server\js
```

Reply:
Please don't use `nvm`, we use conda-native nodejs